.editorconfig

```diff
@@ -1,2 +1,3 @@
 [*.{kt,kts}]
 ktlint_code_style = intellij_idea
+ktlint_standard_no-wildcard-imports = disabled
```

Comment:
Detekt detects it, but more easier to configure. Also we can consider to get rid of ktlint completely. Will create a ticket to backlog.